src/components/dialogs/dataSources/CDC.svelte

```diff
@@ -3,10 +3,11 @@
   import { cdcLocations as regions } from '../../../data/data';
   import SelectField from '../inputs/SelectField.svelte';
   import TextField from '../inputs/TextField.svelte';
+  import { formSelections } from '../../../store';
 
   export let id: string;
 
-  let locations = regions[0].value;
```

Comment:
do we still need this `locations` variable, or can we get rid of it the way we did with the `auth` variables in #72 ?